### PR TITLE
Api setup

### PR DESCRIPTION
**Pull request by JamesAnBo**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+var app = express();
+
+app.get('/api/v1/accounts/:id', (req, res) => {
```
Comment (DylanHul) [inline app.js:4]: Should it be /api/v1/accounts/:id/transactions??

File: app.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  var data = {
+    account: {
+      id: Number(req.params.id)
+    }
+  }
+  console.log(data.account);
+  res.send(data.account)
+})
+
+app.get('/api/v1/accounts/:id/transactions', (req, res) => {
```
Comment (DylanHul) [inline app.js:14]: How is this different from the above route? 👍

File: app.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    account: {
+      id: Number(req.params.id),
+      transactions: []
+    }
+  }
+  console.log(data.account.transactions);
+  res.send(data.account.transactions)
+})
+
+app.listen(3000);
```
Comment (DylanHul) [inline app.js:25]: Would it be nicer to have our api under routes folder? (e.g: routes/transaction.js )

**Comment by DylanHul** [comment]
Can you please review our comments?

File: app.js (hunk 1 above)
Comment (JamesAnBo) [reply app.js:4]: This route is just supposed to return account information. not specific transactions.

File: app.js (hunk 2 above)
Comment (JamesAnBo) [reply app.js:14]: the lower route returns the transactions of the account only.

File: app.js (hunk 3 above)
Comment (JamesAnBo) [reply app.js:25]: yes, but not named transactions. maybe api.js

**Comment by JamesAnBo** [comment]
I'll merge if everyones happy.

**Pull request merged**